examples/ts_js_interop/tsconfig.json

```diff
@@ -0,0 +1,47 @@
+{
+  "compilerOptions": {
+    "emitDeclarationOnly": true,
+    "declaration": true,
+    "jsx": "react",
+    "moduleResolution": "node",
+    "forceConsistentCasingInFileNames": true,
+    "noImplicitReturns": true,
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "experimentalDecorators": true,
+    "lib": ["dom", "es2022", "dom.iterable"],
+    "baseUrl": "./",
+    "paths": {
+      "wkspc/*": ["*", "bazel-out/k8-opt/bin/*"]
```

Comment:
I don't think we should have any `bazel-*` references in this file. Can we clean this up to the minimal config required to reproduce the issue?

Reply:
Do you suspect the tsconfig is the culprit? I figured it was sort of irrelevant- just copied what I had in my project. We can try to trim it down to minimal, but I think this reproduction case hinges on the settings of the ts_project and the swc transpire step.

Reply:
I assume it's irrelevant but would like to remove it to prove that. I'd like the test case to be the absolute minimum amount of code to make it clear and precise what is being tested.

I don't think `paths` is at all related to the bug for example?

Reply:
I modified the tsconfig files to be much more minimal.